### PR TITLE
Eth2 upgrades [wip]

### PR DESCRIPTION
**Pull request by ryancreatescopy**
## Description

Redesign of the ETH2 section.

### CODE REVIEW

**Comment by samajammin** [comment]
FYI @ryancreatescopy there's now the ability to open "draft" PRs. Let's get into the habit of that for WIP PRs:
https://docs.github.com/en/free-pro-team@latest/github/collaborating-with-issues-and-pull-requests/creating-a-pull-request#creating-the-pull-request
https://docs.github.com/en/free-pro-team@latest/github/collaborating-with-issues-and-pull-requests/changing-the-stage-of-a-pull-request#converting-a-pull-request-to-a-draft

I've converted this one already 😄

File: src/components/Warning.js [hunk 1]
```diff
@@ -1,24 +1,32 @@
 import React from "react"
 import styled from "styled-components"
-import Emoji from "./Emoji"
+import { Twemoji } from "react-emoji-render"
```
Comment (samajammin) [inline src/components/Warning.js:3]: FYI @ryancreatescopy - moving forward, we only want to import `Twemoji` in our `<Emoji />`component. Everything else should import our `<Emoji />`component.

**Comment by samajammin** [comment]
@ryancreatescopy I rebased with `dev` & pushed up updates mainly to
1) move the pages under /eth2/ directory
2) add eth2 template to support markdown upgrade pages 
You'll want to run `git reset --hard origin/eth2-v2` before making any further changes in order to avoid conflicts.

File: src/content/eth2/docking/index.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+This extra accessibility will help make Ethereum more secure in the long run. The more people that participate in the network, the more decentralized and secure it will become.
+
+This is also an important change for the second ETH2 upgade: shard chains.
+
+### Setting up for shard chains
+
+Shard chains will be the second Eth2 upgrade. They’ll increase the capacity of network and improve transaction speed by extending the network to 64 blockchains. The docking is an important first step in introducing shard chains, because they require staking to work securely.
+
+Eventually the docking will also be responsible for randomly assigning stakers to validate shard chains. This is key to ensuring stakers can’t become bad guys and collude to take over a shard. Well, it means they have less than a 1 in a trillion chance. This is where the docking’s coordination role comes into play.
```
Comment (lsankar4033) [inline src/content/eth2/docking/index.md:31]: I'd remove 'bad guys' language as protocol incentives are ethically neutral. So maybe something like "This is key to making it difficult for stakers to collude and take over a shard."

File: src/pages/eth2/get-involved.js [hunk 3]
```diff
@@ -0,0 +1,721 @@
...
+      fixed(width: 40) {
+        ...GatsbyImageSharpFixed
+      }
+    }
+  }
+`
+
+export const query = graphql`
+  query {
+    rhino: file(relativePath: { eq: "eth2/eth2_rhino.png" }) {
```
Comment (samajammin) [inline src/pages/eth2/get-involved.js:689]: @ryancreatescopy with the risk of getting too pedantic, let's keep the kebab-case convention for images (& all filenames), i.e. eth2-rino.png

**Comment by samajammin** [comment]
FYI @ryancreatescopy see [this commit](https://github.com/ethereum/ethereum-org-website/pull/1687/commits/3d416b20ca476674097b93093a54d36d7c3c354f) on how to toggle images based on light/dark mode.

File: src/pages/eth2/get-involved/index.js [hunk 4]
```diff
@@ -0,0 +1,507 @@
...
+  align-items: center;
+  margin-top: 2rem;
+`
+
+const StyledCardContainer = styled(CardContainer)`
+  margin-top: 2rem;
+  margin-bottom: 3rem;
+`
+
+const H2 = styled.h2`
```
Comment (samajammin) [inline src/pages/eth2/get-involved/index.js:101]: @ryancreatescopy I see we re-use these H2 styles a bunch - can we move this into SharedStyledComponents? Should this replace our existing H2 already there?

**Review by samajammin — DISMISSED** [review]
LGMT @ryancreatescopy please wait til build succeeds & confirm all looks good before merging.

**Review by jjmstark — APPROVED** [review]

**Pull request merged**